gigascience/gigadb-website
# Remove link to dimensions

# Pull request for issue: #2008

removed link to Dimensions from dataset view

## How to test?

- go to http://gigadb.gigasciencejournal.com/dataset/100094
- verify that dimensions link is not present as mentioned in issue

## How have functionalities been implemented?

Removed the dimensions values from the file where they originated, no actual changes in the view

## Any issues with implementation?

this test is failing, but unrelated to this work
![image](https://github.com/user-attachments/assets/835496a9-156b-46c3-b5ef-38f7500c64f2)

## Any changes to automated tests?

- Needed to update / remove the tests that expected the dimension link to be present
